flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStResourceContainer.java

```diff
@@ -190,7 +191,10 @@ public DBOptions getDbOptions() {
         // configured,
         //  fallback to local directory currently temporarily.
         if (remoteForStPath != null) {
-            flinkEnv = new FlinkEnv(remoteForStPath.toString(), forstFileSystem);
+            flinkEnv =
+                    new FlinkEnv(
+                            remoteForStPath.toString(),
+                            new StringifiedForStFileSystem(forstFileSystem));
```

Comment:
I am not very familiar with this code. I wonder if forstFileSystem can be created stringified, or is there a need for it to be around in a non-stringified form?